src/builtins/core/duration.rs

```diff
@@ -353,12 +353,6 @@ impl Duration {
         &self.date
     }
 
-    /// Set this `DurationRecord`'s `TimeDuration`.
-    #[inline]
-    pub fn set_time_duration(&mut self, time: TimeDuration) {
-        self.time = time;
-    }
-
```

Comment:
Is this removal intended?

Reply:
Removal is intended primarily due to [this comment](https://github.com/boa-dev/temporal/blob/main/temporal_capi/src/duration.rs#L166)